### PR TITLE
api week - change day 2 to include flexbox froggy & put modular design workshop back in

### PR DESCRIPTION
**Pull request by lucyrose93**
Updated Week 3 schedule to reflect the changes we have made to Day Two 
+ add flexbox froggy
+ add @smarthutza's flexbox workshop
+ put @eliascodes' workshop back in, instead of research afternoon
+ add time at the end of the day for students to begin project by implementing software design ideas they have just learned

relates #300

### CODE REVIEW

**Comment by jsms90** [comment]
@lucyrose93 Just updated the body of your PR. Please try to be as descriptive as possible

**Comment by jsms90** [comment]
@eliascodes @des-des @bradreeder Emergency merge :sweat:

File: coursebook/week-3/README.md [hunk 1]
```diff
@@ -26,25 +26,20 @@
...
-- 14.00 - 15.00 <br /> 
-Software Architecture continues
-
-- 15.00 - 16.30 <br /> 
-[Research](link to be updated)
-
-- 16.30 - 17.30 <br /> 
-Presentations 
+- 14.00 - 17.00 <br />
+[Software Design Workshop]()
```
Comment (jsms90) [inline coursebook/week-3/README.md]: @lucyrose93 Just seen this doesn't go anywhere

**Comment by mattlub** [comment]
@jsms90 what happens now this was 'emergency merged'? I don't really understand why you can't just direct the students to a different branch or tell the students the schedule has changed slightly, if there isn't enough time to review changes.

**Comment by jsms90** [comment]
@mattlub We fixed it in [this PR](https://github.com/foundersandcoders/master-reference/pull/457)

In the contributing guidelines' [instructions for maintainers](https://github.com/foundersandcoders/master-reference/blob/master/CONTRIBUTING.md#instructions-for-maintainers), it does state
> Once a pull request has been approved by two maintainers it can be merged. In time critical situations, one approval may suffice, as long a the pull request is small and is not suggesting any major change.

I completely agree that this should not really be happening. In this particular instance, it is just a reflection of the fact that London's week 3 mentors had not finished planning their week prior to arriving on day 1.

As Course Facilitators we should be aiming for this to happen as little as possible
> I don't really understand why you can't just direct the students to a different branch

Pointing students to unmerged content is essentially the same as an emergency merge, only there is no online record of what has _actually_ been taught.

**Comment by mattlub** [comment]
@jsms90 yes sorry, I didn't doubt you were following the guidelines. I guess I just disagree on this: 
> pointing students to unmerged content is essentially the same as an emergency merge

Leaving it unmerged means there will be a discussion on whether or not it should be merged and/or changed, whereas merging it implies it has been approved and so a new discussion would have to be raised to change it.

It's not a massive deal, you do a great job maintaining the master ref, it's just there was literally a discussion going on about this at the time it got merged.

**Comment by jsms90** [comment]
Yeah I totally get what you're saying. A conversation should really be going on prior to merging. This scenario is very very far from ideal.

But leaving it unmerged and then having a conversation about what _should_ have been done _after_ the curriculum has actually been delivered, means that you're always playing catch up and documenting after the fact.

This is what I'm aiming for:
+ Before a week is delivered, we merge in what is about to be delivered to the students.
+ During the week, when students, mentors and facilitators think of potential improvements, issues are raised.
+ During / after the week, every member of FAC is then able and encouraged to comment and discuss as a community, prior to curriculum planning week
+ During curriculum planning, students divide themselves between the weeks and planning start with each set of mentors looking through an existing backlog of issues.

**Pull request merged**